sdk/src/clearingHouseUser.ts

```diff
@@ -568,6 +572,11 @@ export class ClearingHouseUser {
 				const market = this.clearingHouse.getMarketAccount(
 					marketPosition.marketIndex
 				);
+
+				if (market === undefined) {
```

Comment:
dont like this, think it should throw an error if the market isnt there. could see this leading to nasty bugs

Reply:
did this since functions break if theres no market at marketIndex=0 in a new test written

way off for this pr but since its possible to delist a market, feel like we should handle that better

Reply:
reverting it here and going to fix test